src/site/content/en/blog/lighthouse-user-flows/index.md

```diff
@@ -0,0 +1,250 @@
+---
+title: 'Lighthouse user flows'
+subhead: Try out a new Lighthouse API to measure throughout a user flow.
+description: Try out a new Lighthouse API to measure throughout a user flow.
```

Comment:
Is there a word missing in these? "To measure [potentially missed word] through a user flow?"